src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -92,6 +95,9 @@ public ForwardGenerator(
     this.instantiator = componentManager.getTypeInstantiator();
 
     initializeRuntimePrimitivesSeen();
+
+    // Copy all operations into bloodhound's list of operations.
+    bloodhound.setOperations(operations);
```

Comment:
For better encapsulation, I suggest doing no initialization on line 73, and `bloodhound = new Bloodhound(operations);` here.